tests/foreman/cli/test_user.py

```diff
@@ -106,10 +105,7 @@ def test_bugzilla_1079649_2(self):
 
     @data(
         {'login': gen_string("latin1", 10)},
-        {
-            'login': gen_string("utf8", 10),
-            'bugzilla': 1144162,
-        },
+        {'login': gen_string("utf8", 10)},
```

Comment:
not a blocker, but we can drop the 10's out of this section, since 10 is the default.